servers/lib/src/libms.command.ts

```diff
@@ -0,0 +1,27 @@
+import { Command, CommandRunner, Option } from 'nest-commander';
+import { readFileSync} from 'fs';
+import { parse } from 'dotenv';
+
+@Command({
+    name: 'libms'
+})
+
+export default class LibmsCommand extends CommandRunner {
+    private configFile: string;
+
+    async run(_inputs: string[], options: Record<string, string>): Promise<void> 
+    {
+        if (options.config) {
+            this.parseConfigFile(options.config);
+        }
+        const config = readFileSync(this.configFile);
+        parse(config);
+    }
+    @Option({
+        flags: '-c, --config <config-file>',
```

Comment:
adding `-h` / `--help` flag will provide a display of config options and a small help text.

Help text:

The lib microservice is responsible for handling and serving the contents of library assets of the DTaaS platform. It provides API endpoints for clients to query, and fetch these assets.